crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -915,6 +923,22 @@ impl Tetrahedron {
     pub fn centroid(&self) -> Vec3 {
         (self.vertices[0] + self.vertices[1] + self.vertices[2] + self.vertices[3]) / 4.0
     }
+
+    /// Get the triangles that form the faces of this tetrahedron.
+    ///
+    /// Note that the orientations of the faces are determined by that of the tetrahedron; if the
+    /// signed volume of this tetrahedron is positive, then the triangles' normals will point
+    /// outward, and if the signed volume is negative they will point inward.
+    #[inline(always)]
+    pub fn faces(&self) -> [Triangle3d; 4] {
+        let [a, b, c, d] = self.vertices;
+        [
+            Triangle3d::new(b, c, d),
+            Triangle3d::new(a, c, d).reversed(),
```

Comment:
Hmm, I guess so, but I thought this would make it clearer what's actually going on, since the relationship between vertex order and orientation is otherwise not that clear (e.g. the choice of which vertices to interchange when `Triangle3d::reverse` is called is essentially arbitrary).

Reply:
Yeah, this is clearer to me.